src/main/java/org/reaktivity/nukleus/sse/internal/stream/SseServerFactory.java

```diff
@@ -972,37 +968,57 @@ private void doFlush(
                 }
             }
 
-            if (networkSlot != NO_SLOT)
+            if (deferredCredit > 0)
             {
-                final MutableDirectBuffer buffer = bufferPool.buffer(networkSlot);
-                final DataFW data = dataRO.wrap(buffer,  0,  networkSlotOffset);
-                final int networkReplyDebit = data.reserved();
+                final int reserved = deferredCredit + networkReplyPadding;
```

Comment:
`deferredClaim` should already include reply padding, so `reserved = deferredClaim`.
In fact, no need for separate `reserved` variable, can directly use `deferredClaim` instead.